Automattic/woocommerce-payments
# Update Affirm & Afterpay logos with border and icon images

Fixes #6563 

#### Changes proposed in this Pull Request
Update logos:

1. Replace Affirm logo with grey border in it. 
2. Use icon logos within Payment list and payment details view. 



#### Testing instructions 
1. Make a purchase with each BNPL provider (Affirm & Afterpay)
2. Go to Payments > Transactions and make sure the logo is displayed correctly.
3. Go to transaction detail and make sure the logo is displayed correctly.





*

-------------------


**Post merge**
